packages/component/src/Composer.js

```diff
@@ -93,6 +93,8 @@ function createCardActionContext({ cardActionMiddleware, directLine, dispatch })
                   return observableToPromise(directLine.getSessionId()).then(
                     sessionId => `${value}${encodeURIComponent(`&code_challenge=${sessionId}`)}`
                   );
+                } else {
+                  console.warn('botframework-webchat: OAuth is not supported on this Direct Line adapter.');
```

Comment:
Should we add a link here? 

`https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication`